scripts/linters/pylint_extensions_test.py

```diff
@@ -2408,6 +2426,146 @@ def test_type_ignore_used_without_comment_raises_error(self):
             )
         temp_file.close()
 
+    def test_raises_error_if_prohibited_error_code_is_used(self):
```

Comment:
Should you have tests for "no error if allowed error code is used in single/combined form", or is that not necessary?

Reply:
> no error if allowed error code is used in single/combined form

If an allowed error code is used then it should be documented by a code comment -- So, a test for combined type_ignore is added ( which was missing previously ).